core/modules/SConscript

```diff
@@ -201,6 +201,8 @@ if not env.GetOption('clean'):
 
     env.Append(CPPPATH=[os.getcwd()])
 
+    env.Append(CCFLAGS = ' -Og ')
```

Comment:
You don't actually need spaces; SCons takes care of separating all CCFLAGS entries.

Reply:
just a typo but thx for catching.